Phonebook.Frontend/.gitignore

```diff
@@ -1,47 +1,47 @@
-# See http://help.github.com/ignore-files/ for more about ignoring files.
-
-# compiled output
-dist/
-/tmp
-/out-tsc
-/src/changelog.md
-
-# dependencies
-node_modules/
-
-# IDEs and editors
-/.idea
-.project
-.classpath
-.c9/
-*.launch
-.settings/
-*.sublime-workspace
-
-# IDE - VSCode
-.vscode/*
-!.vscode/*.code-snippets
-!.vscode/settings.json
-!.vscode/tasks.json
-!.vscode/launch.json
-!.vscode/extensions.json
-
-# misc
-/.sass-cache
-/connect.lock
-/coverage
-/libpeerconnection.log
-npm-debug.log
-testem.log
-/typings
-yarn-error.log
-key.pem
-cert.pem
-
-# e2e
-/e2e/*.js
-/e2e/*.map
-
-# System Files
-.DS_Store
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
This file should not be committed again as we solved this issue in the other PR. Please rebase or squash the commits in this PR and force push again.